agent/common.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -euxo pipefail
+set -o pipefail
```

Comment:
This change appears to disable logging, as well as allow undefined variables and not exit on error. I don't think we want that. Was this accidentally left over from some debugging?

Reply:
Yeah now I remember, I changed that because there was a bug in including validation.sh that I thought I'd fix later. Thanks for pointing it out. Fixed.